src/nucleus/value.ml

```diff
@@ -494,6 +494,16 @@ let print_value'' env ?max_level v ppf =
   Format.fprintf ppf "@[<hov>%t@]"
                  (print_value' ?max_level ~penv refs v)
 
+let print_operation env op vs ppf =
+  if vs = []
+  then Name.print_ident op ppf
+  else
+    let penv = get_penv env
+    and refs = env.state in
+    Format.fprintf ppf "@[<hov>%t@ %t@]"
+      (Name.print_ident op)
+      (Print.sequence (print_value' ~max_level:0 ~penv refs) "" vs)
```

Comment:
I think the correct level here is `Tt.Lvl.prefix`, but we do not expose that. I'll mege this and make an issue about that.